examples/kubernetes/lib/kube.libjsonnet

```diff
@@ -171,13 +173,26 @@ local inv = kap.inventory();
 
     imagePullSecrets: [],
 
+    nodeSelector: {
+      [if "nodepools" in inv.parameters.cluster && "nodeselector" in inv.parameters then inv.parameters.cluster.nodepools[inv.parameters.nodeselector].label]: inv.parameters.cluster.nodepools[inv.parameters.nodeselector].name,
```

Comment:
This is specific to us, not sure other people might want this

Reply:
I agree